internal/utils.go

```diff
@@ -157,8 +158,9 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]map[string
 			}
 
 			objTemplate := map[string]interface{}{
-				"complianceType":   complianceType,
-				"objectDefinition": manifest,
+				"complianceType":         complianceType,
+				"metadataComplianceType": metadataComplianceType,
```

Comment:
Could you only set this if `metadataComplianceType != ""`? That way the generated policy is a little cleaner when this value isn't set.